lib/designation.js

```diff
@@ -40,6 +40,7 @@ var DEFAULT_FILTER_DOCKER_NFS_VOLUMES_AUTOMOUNT_MIN_PLATFORM =
     '20160613T123039Z';
 var DEFAULT_FILTER_NON_DOCKER_NFS_VOLUMES_AUTOMOUNT_MIN_PLATFORM =
     '20170925T211846Z';
+var DEFAULT_FILTER_VOLAPI_NFS_V2_MIN_PLATFORM = '20191106T000000Z';
```

Comment:
This will need to be updated once Dan commits the NFS platform changes.

Reply:
Still pending? Also, shouldn't the (C) notice be modified for the file?

Reply:
Updated now.